dist/templates/k8s.cni.cncf.io_ipamclaims.yaml

```diff
@@ -0,0 +1,63 @@
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
we don't want to have a local copy for this intentionally right?
I remember @jcaamano / @maiqueb we decided we can just pull and oc apply from remote...

Reply:
Hi @maiqueb !

Hmm... I did not introduce the file "explicitly"... It was added by `update-codegen.sh`. If the intention is to not do that, then we need to fix the script so this does not become an ongoing behavior. Could you please take a look?
All I did was to invoke `update-codegen.sh` as described in the commit message.

Thanks,

-- flaviof